private/buf/cmd/buf/buf.go

```diff
@@ -199,7 +197,6 @@ func NewRootCommand(name string) *appcmd.Command {
 								Use:   "plugin",
 								Short: "Manage Protobuf plugins",
 								SubCommands: []*appcmd.Command{
-									plugincreate.NewCommand("create", builder),
```

Comment:
From the CLI side, can't we remove this entire block under `plugin`?

Reply:
Removed - PTAL.